frontend/public/locales/en/analytics.json

```diff
@@ -2,7 +2,15 @@
   "analytics": "Analytics",
   "dashboard": {
     "title": "Dashboards",
-    "description": "This page lists all the dashboards."
+    "description": "This page lists all the dashboards.",
```

Comment:
No Chinese in zh i18n` /zh/analytics.json` ?

Reply:
done